esx_service/vmdk_ops.py

```diff
@@ -648,7 +653,9 @@ def disk_attach(vmdk_path, vm):
                      if type(dev) == vim.VirtualDisk and dev.controllerKey ==
                      controller_key])
         # search in 15 slots, with unit_number 7 reserved for scsi controller
-        availSlots = set(range(0, 6) + range(8, 16)) - taken
+        range1 = set(range(0, 7))
```

Comment:
why do we need 3 lines instad of one now ? I understand  you fixed a bug (Missing slot #6), but why not simply leave it as 
```python
availSlots = set(range(0, 7) + range(8, 16)) - taken
``` 
?